dashboard/app/views/weblab_host/index.html.haml

```diff
@@ -1,6 +1,8 @@
 %html{spellcheck: 'false'}
   %head
+    = javascript_include_tag 'application'
```

Comment:
What changed to make this necessary?  Or did we need this all along and just forgot it?